src/test/java/org/apache/drill_web_test_framework/rest_api/tests/secure/ProfilesTest.java

```diff
@@ -30,6 +36,19 @@
 import static org.hamcrest.Matchers.hasItems;
 
 public class ProfilesTest extends BaseRestTest {
+  private static String bigQuery = null;
+
+  @BeforeClass
+  public static void getLargeQuery() {
+    StringBuffer sqlQuery = new StringBuffer();
```

Comment:
There is already a method in RestStorageSteps:
`private static String getPluginFromResource(String resource)`
I think it should by moved to RestBaseSteps, made public and renamed to something like getStringFromResource.

Reply:
Done.